srcpkgs/jalv/template

```diff
@@ -0,0 +1,25 @@
+# Template file for 'jalv'
+pkgname=jalv
+version=1.6.6
+revision=1
+build_style=waf3
+configure_args="$(vopt_if portaudio '--portaudio') --no-gtkmm --no-gtk2
+ $(vopt_if gtk3 '' '--no-gtk3') $(vopt_if qt '' '--no-qt5')"
+hostmakedepends="pkg-config $(vopt_if qt qt5-host-tools)"
+makedepends="lv2 lilv-devel suil-devel serd-devel sord-devel sratom-devel
+ $(vopt_if portaudio portaudio-devel jack-devel) $(vopt_if gtk3 gtk+3-devel)
+ $(vopt_if qt qt5-devel)"
+short_desc="LV2 host for JACK"
+maintainer="tibequadorian <tibequadorian@posteo.de>"
+license="GPL-3.0-or-later"
```

Comment:
Should be ISC https://gitlab.com/drobilla/jalv/-/blob/master/COPYING

Reply:
oh :sweat_smile: sorry